.cargo/config

```diff
@@ -1,3 +1,26 @@
+# NOTE: if you make changes here, remember to also update:
+#	scripts/test-linux.sh
+#	scripts/build-linux.sh
+#	scripts/build-windows.sh
+
+# Using 'cfg` is broken, see https://github.com/rust-lang/cargo/issues/6858
+#[target.'cfg(target_arch = "x86_64")']
+#rustflags = ["-Ctarget-feature=+aes,+sse2,+ssse3"]
+
+# …so instead we list all target triples (Tier 1 64-bit platforms)
+[target.x86_64-unknown-linux-gnu]
+# Enable aesimc. 
```

Comment:
Small nit, easimc is one of the instruction resulting from eas-ni activation, so I would prefer a comment along the line of "Enable aes-ni instruction for RustCrypto dependency"